boot/zephyr/Kconfig.serial_recovery

```diff
@@ -155,6 +164,29 @@ config BOOT_SERIAL_WAIT_FOR_DFU_TIMEOUT
 	default 500
 	depends on BOOT_SERIAL_WAIT_FOR_DFU
 	help
-	  timeout in ms for MCUboot to wait to allow for DFU to be invoked.
+	  Timeout in ms for MCUboot to wait to allow for DFU to be invoked.
+
+config BOOT_SERIAL_BOOT_MODE
+	bool "Check boot mode via retention subsystem"
+	depends on RETENTION_BOOT_MODE
+	help
+	  Allows for entering serial recovery mode by using Zephyr's boot mode
+	  retention system (i.e. an application must set the boot mode to stay
+	  in serial recovery mode and reboot the module).
```

Comment:
What is module?

Reply:
As in what is the retention system? Or another question?

Reply:
I was referring to module in `... reboot the module`. How about `device` or `SoC`?